lib/handle_flags.js

```diff
@@ -0,0 +1,26 @@
+const flags = require('flags');
+
+flags.defineBoolean('test')
+    .setDefault(false)
+    .setDescription('Use this flag to print information in the console.');
+
+flags.defineString('channel')
+    .setDefault('web_ids_requests')
+    .setDescription('Redis channel to listen to.' +
+                    ' Channel must be JSON formatted.');
+
+flags.defineBoolean('local')
```

Comment:
`local` is not that clear for me. maybe change it to `dev` instead and adapt the description accordingly.
The `local` flag is a replacement for the development environment, that is why `dev` might make more sense in the code

Reply:
I agree

Reply:
I agree, will be corrected accordingly